test/NuGet.Core.Tests/NuGet.Common.Test/PathResolverTests.cs

```diff
@@ -664,17 +678,20 @@ public void Dispose()
         private void PopulateTestDirectory()
         {
             var rootDirectory = new DirectoryInfo(_rootDirectory.Path);
-            DirectoryInfo directory1 = Directory.CreateDirectory(System.IO.Path.Combine(rootDirectory.FullName, "dir1"));
-            DirectoryInfo directory2 = Directory.CreateDirectory(System.IO.Path.Combine(directory1.FullName, "dir2"));
-            DirectoryInfo directory3 = Directory.CreateDirectory(System.IO.Path.Combine(directory2.FullName, "dir3"));
-            DirectoryInfo directory4 = Directory.CreateDirectory(System.IO.Path.Combine(directory1.FullName, "dir4"));
-            Directory.CreateDirectory(System.IO.Path.Combine(rootDirectory.FullName, "dir5"));
+            var directory1 = Directory.CreateDirectory(System.IO.Path.Combine(rootDirectory.FullName, "dir1"));
+            var directory2 = Directory.CreateDirectory(System.IO.Path.Combine(directory1.FullName, "dir2"));
+            var directory3 = Directory.CreateDirectory(System.IO.Path.Combine(directory2.FullName, "dir3"));
+            var directory4 = Directory.CreateDirectory(System.IO.Path.Combine(directory1.FullName, "dir4"));
+            var directory5 = Directory.CreateDirectory(System.IO.Path.Combine(rootDirectory.FullName, "dir5"));
+            var directory6 = Directory.CreateDirectory(System.IO.Path.Combine(rootDirectory.FullName, "dir6"));
```

Comment:
`System.IO` is redundant because this namespace is mentioned in the `using` statement on top of the file.

Reply:
There's a clash with:

https://github.com/NuGet/NuGet.Client/blob/d39e857e6b1265d1b4990cbe660e2a0ad259e0cf/test/NuGet.Core.Tests/NuGet.Common.Test/PathResolverTests.cs#L650

To remove those namespace declarations, either

1) Refactor that variable name and introduce lots of changes.
2) Leave it as it is.